src/assets/lang/es/sustainability.json

```diff
@@ -0,0 +1,97 @@
+      },
+      "element2": {
+        "title": "Mayores ahorros",
+        "description": "Solo utilizamos centros de datos energéticamente eficientes que minimizan las necesidades de hardware, reducen los gastos operativos y aumentan el ahorro de costos de servicios públicos, lo que se alinea con nuestros objetivos de desarrollo verdes y sostenibles."
+      },
+      "element3": {
+        "title": "Eficiencia",
+        "description": "Nuestros centros de datos cuentan con la certificación ISO50001 y participan en el Código Europeo de Conducta para la Eficiencia Energética en Centros de Datos, el estándar reconocido internacionalmente para la gestión energética."
+      },
+      "element4": {
+        "title": "Capital Verde Europea 2024",
+        "description": "Con sede en Valencia, Internxt se inspira en los esfuerzos de sostenibilidad de la ciudad, impulsando nuestras propias iniciativas ecológicas a través de soluciones ecológicas de almacenamiento en la nube."
+      }
+    }
+  },
+  "FeatureSectionV3": {
+    "title": "Sostenibilidad en números",
+    "description": "Internxt lidera con el ejemplo trabajando por un futuro más sostenible a través de prácticas de eficiencia energética para minimizar el consumo de recursos y contribuir a un futuro más verde y sostenible.",
+    "cards": {
+      "power": {
+        "title": "Energía",
+        "description": "La eficacia del uso de energía (PUE) de nuestros centros de datos se mide y audita cada 12 meses. Los valores medidos más cerca de 1 indican un consumo de energía más eficiente.",
+        "value": "1.26",
+        "unit": "PUE"
+      },
+      "water": {
+        "title": "Agua",
+        "description": "Esta métrica mide la eficiencia con la que se utiliza el agua; los valores más cercanos a 0 indican un rendimiento óptimo.",
+        "value": "0.37",
+        "unit": "L/kWh IT"
+      },
+      "carbon": {
+        "title": "Carbón",
+        "description": "Este rastrea las emisiones directas y las comisiones indirectas del uso de electricidad, donde los valores más cercanos a 0 representan un mejor rendimiento.",
+        "value": "0.16",
+        "unit": "Kg C02e / kWh IT"
+      },
+      "renewables": {
+        "title": "Renovables",
+        "description": "Esto se refiere al porcentaje de tarifas de Factor de Energía Renovable (REF) utilizadas en nuestros centros de datos.",
+        "value": "92%",
+        "unit": "REF"
+      }
+    }
+  },
+  "FeatureSection": {
+    "title": "Cómo nos comprometemos con un futuro sostenible",
+    "description": "Al promover iniciativas ecológicas, nuestro objetivo es proteger el planeta y esforzarnos por lograr una solución de almacenamiento en la nube más limpia y ecológica para 2025.",
+    "cards": {
+      "reduceCarbonEnergy": {
+        "number": "1",
+        "title": "Reduce carbon energy",
```

Comment:
This ones are in english and not in spanish

Reply:
Updated